src/containers/App.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react'
 import { GlobalStyle } from "../components/styles/globals";
+import SearchResults from '../pages/SearchResults';
 
 function App() {
   return (
     <div className="App">
       <GlobalStyle />
-      <h1>Music App</h1>
+      <SearchResults></SearchResults>
```

Comment:
Return <h1> Music App</h1>   and see the video from Oscar

Reply:
Below you can see my fix to undo changes in App.js